README.md

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
Please add instructions to set up the project locally. It is important for a non-technical user in order to use the project.